src/contracts/Translation/AbstractTranslationCase.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Contracts\Test\Core\Translation;
+
+use Ibexa\Contracts\Test\Core\IbexaKernelTestCase;
+use JMS\TranslationBundle\Model\Message;
+
+abstract class AbstractTranslationCase extends IbexaKernelTestCase
+{
+    abstract protected function getConfigName(): string;
+
+    final public function testTranslation(): void
+    {
+        $facade = $this->getTranslationService();
+        $changeset = $facade->getChangeSet($this->getConfigName());
```

Comment:
How about making it a data provider? That way you could also test more of them if needed.